config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -249,6 +249,39 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-gcp
       description: "Run e2e test with gcp provider for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-aws
+    decorate: true
+    decoration_config:
+      timeout: 25m
+    always_run: true
+    optional: true
+    path_alias: sigs.k8s.io/secrets-store-csi-driver
+    branches:
+    - master
+    labels:
+      # this is required because we want to run kind in docker
+      preset-dind-enabled: "true"
+      # this is required to make CNI installation to succeed for kind
+      preset-kind-volume-mounts: "true"
+      # Sets up the aws parameters used for testing
+      preset-aws-credential-aws-oss-testing: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210509-00de27f-master
+        command:
+        - runner.sh
+        args:
+        - bash
+        - -c
+        - >-
+          make e2e-bootstrap e2e-helm-deploy e2e-aws
```

Comment:
1. `e2e-bootstrap` is for setting up the kind cluster, building the image and loading the docker image into the cluster. Looking at https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/533/files#diff-7e3f8456f53d3bc514e079d8c2fb31a1b9d5f66ec579d4a6d2987829524f6625R21-R37, it might not be relevant for aws?
2. `e2e-helm-deploy` deploys the driver with the built image in the cluster. But since the cluster is being deployed as part of the bats suite, it's not required.

Reply:
Will remove e2e-bootstrap and introduce new step, setup-eks-cluster that will create and initialize the test cluster